packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -187,9 +187,13 @@ export class PluginLoader {
           this._plugins.push(plugin);
           // Enable each supported plugin.
           return plugin.enable(exports, this.provider, this.logger, config);
-        } catch (e) {
-          this.logger.error(
-            `PluginLoader#load: could not load plugin ${modulePath} of module ${name}. Error: ${e.message}`
+        } catch (err) {
+          const logLevel =
+            err instanceof Error && err.message.match(/Cannot find module/)
```

Comment:
Would it be better to match on `error.code` which is a more stable error identifier?

https://nodejs.org/api/errors.html#errors_error_code

https://nodejs.org/api/errors.html#errors_err_module_not_found